### PR TITLE
DOC Fix discussions of frontends

### LINKED ISSUE #583: Include Keras, sklearn in docs and structure
Right now, it's all one big text. We should split up different subsections for different frontends. At the top, we should have a quick text summarizing the frontend framework and describing how to import them dynamically (using `kymatio.Scattering*D` and specifying `frontend` when creating).

### PR DESCRIPTION
**Pull request by janden**
Fixes #583.

### CODE REVIEW

**Comment by janden** [comment]
Starting to feel a bit repetitive here (similar code samples appear in a few places now: README, user guide, code reference). Thinking of moving the text from the code reference to user guide so we remove that duplication, at least. What do you think?

**Comment by janden** [comment]
# [Codecov](https://codecov.io/gh/kymatio/kymatio/pull/589?src=pr&el=h1) Report
> Merging [#589](https://codecov.io/gh/kymatio/kymatio/pull/589?src=pr&el=desc) into [dev](https://codecov.io/gh/kymatio/kymatio/commit/2a2f010616bf45b85d476fb453b85bf86024937e&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

[Impacted file tree graph](https://codecov.io/gh/kymatio/kymatio/pull/589?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##              dev     #589   +/-   ##
=======================================
  Coverage   87.70%   87.70%           
=======================================
  Files          65       65           
  Lines        2342     2342           
=======================================
  Hits         2054     2054           
  Misses        288      288           
```

| Flag | Coverage Δ | |
|---|---|---|
| #jenkins_main | `87.70% <ø> (ø)` | |
| #travis | `77.79% <ø> (ø)` | |


------

[Continue to review full report at Codecov](https://codecov.io/gh/kymatio/kymatio/pull/589?src=pr&el=continue).
> **Legend** - [Click here to learn more](https://docs.codecov.io/docs/codecov-delta)
> `Δ = absolute <relative> (impact)`, `ø = not affected`, `? = missing data`
> Powered by [Codecov](https://codecov.io/gh/kymatio/kymatio/pull/589?src=pr&el=footer). Last update [2a2f010...cecfa06](https://codecov.io/gh/kymatio/kymatio/pull/589?src=pr&el=lastupdated). Read the [comment docs](https://docs.codecov.io/docs/pull-request-comments).

**Comment by edouardoyallon** [comment]
> Starting to feel a bit repetitive here (similar code samples appear in a few places now: README, user guide, code reference). Thinking of moving the text from the code reference to user guide so we remove that duplication, at least. What do you think?

It's a good idea. Repetition is in general bad.

**Comment by janden** [comment]
> It's a good idea. Repetition is in general bad.

Ok will take a look later.

**Comment by janden** [comment]
Ok. Qu'en pensez vous, M.?

File: README.md [hunk 1]
```diff
@@ -84,41 +84,46 @@ from kymatio.numpy import Scattering2D
...
 ```
 
+## Scikit-learn
+
+After installing the latest version of scikit-learn, you can call `Scattering2D` as a `Transformer` using:
+
+```
+from kymatio.sklearn import Scattering2D
+
+scat = Scattering2D(2, (32, 32))
```
Comment (eickenberg) [inline README.md]: naming is meh

**Pull request merged**